book/src/csi-driver-object.md

```diff
@@ -45,6 +48,13 @@ There are three important fields:
     * `"csi.storage.k8s.io/pod.namespace": pod.Namespace`
     * `"csi.storage.k8s.io/pod.uid": string(pod.UID)`
   * For more information see [Pod Info on Mount](pod-info.md).
+* `volumeLifecycleModes`
+  * This field was added in Kubernetes 1.16 and cannot be set when using an older Kubernetes release.
+  * It informs Kubernetes about the volume modes that are supported by the driver.
+    This ensures that the driver [is not used incorrectly](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190122-csi-inline-volumes.md#support-for-inline-csi-volumes) by Kubernetes.
```

Comment:
"by Kubernetes" or "by users"?

Reply:
"by Kubernetes on behalf of a user". That's becoming complicated - perhaps the sentence should simply end with "is not used incorrectly".

Reply:
I think "by users" is fine

Reply:
"by Kubernetes" implies to me that we're protecting against Kubernetes bugs, which I don't think is the case here

Reply:
Changed to "by users".